docs/UserGuide.md

```diff
@@ -38,9 +38,11 @@ It has the benefits of a Graphical User Interface (GUI). If you can type fast, N
    * `list` : Lists all contacts.
 
    * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
-   
+
+   * `addBuy 1 ht/c bp/1650000 pc/567510 un/10-65 t/Extremely spacious t/Near MRT` : Adds a property of type `Condo` to the Address Book for the contact at index 1.
+
```

Comment:
I think you can further specify, that this property is being added to the list of properties the contact wants to buy
=> Adds a property of type 'Condo' to the Address Book for the contact under the contact's list of Properties to buy at index 1